cmake/ConfigFileSetting.cmake

```diff
@@ -46,12 +46,18 @@ set(libdir \${exec_prefix}/lib)
 set(USING_CMAKE_FALSE "#")
 set(USING_CMAKE_TRUE "")
 
-# ~~~
-# TODO : these two don't start out as #undef but as #define so need their
-# explicit @...@ replacments
-# set(NEURON_BIN_DIR "\"${CMAKE_INSTALL_PREFIX}/${CMAKE_SYSTEM_PROCESSOR}/bin\"")
-# set(NRN_CONFIG_ARGS "\"unknown\"") # ends up as "" since no @...@
-# ~~~
+# A variable that doesn't start out as #undef but as #define needs an
```

Comment:
```suggestion
# ~~~
# A variable that doesn't start out as #undef but as #define needs an
```

I will keep the guard `# ~~~` so that when we use cmake-format to format the CMake files, it will the comment block intact.

Reply:
Sorry. I forgot what those were. A very useful idiom.